srtcore/queue.cpp

```diff
@@ -970,6 +972,11 @@ void CRendezvousQueue::updateConnStatus(EReadStatus rst, EConnectStatus cst, con
                 i->m_pUDT->m_RejectReason = SRT_REJ_PEER;
             }
             CUDT::s_UDTUnited.m_EPoll.update_events(i->m_iID, i->m_pUDT->m_sPollID, SRT_EPOLL_ERR, true);
+            if (i->m_pUDT->m_cbConnectHook)
+            {
+                CALLBACK_CALL(i->m_pUDT->m_cbConnectHook, i->m_iID,
+                        is_timeout ? SRT_ENOSERVER : SRT_ECONNREJ, i->m_PeerAddr.get());
```

Comment:
`is_timeout` variable is not necessary.

```suggestion
                CALLBACK_CALL(i->m_pUDT->m_cbConnectHook, i->m_iID,
                        i->m_pUDT->m_RejectReason == SRT_REJ_TIMEOUT ? SRT_ENOSERVER : SRT_ECONNREJ, i->m_PeerAddr.get());
```